src/main/java/org/spongepowered/api/entity/player/User.java

```diff
@@ -85,25 +87,27 @@
     Date getLastPlayed();
 
     /**
-     * Checks if this player is banned.
+     * Checks if this player is online or not.
      *
-     * @return True If banned
+     * @return True if the corresponding player is online
      */
-    boolean isBanned();
+    boolean isOnline();
 
     /**
-     * Checks if this player is whitelisted.
+     * Gets the location of this player's bed spawn.
      *
-     * @return True If whitelisted
+     * @return The location of this player's bed spawn, or
+     * {@link Optional#absent} if it is not set
      */
-    boolean isWhitelisted();
```

Comment:
And by removing them here you will only cause conflicts when that PR is merged in, better to leave those methods untouched for gratimax's branch to change.